rfc-brief.Rmd

```diff
@@ -0,0 +1,225 @@
+Lastly, journal requirements and other repositories also had some influence on these specifications.
+
+
+## Data types and formats summary
+
+Table \@ref(tab:table-main) summarizes specifications for common data types that are of interest for the NF community.
+These standards should apply for most general cases.
+However, we do note that for some data types, the community comments provided additional considerations that serve to qualify or supplement what is in this table.
+Community results are also summarized in Quantified Results section. 
+
+```{r table-main, fig.label="", layout="l-page", echo=F}
+
+# for gt, use captions instead of fig.label
+table1 <- read.csv("materials/rfc/data_specs.csv")
+table1 %>%
+  select(-c(Description, PrivateNotes)) %>%
+  gt(
+  rowname_col = "Subtype",
+  groupname_col = "ParentType",
+  caption = "Specifications for relevant common data types."
+  ) %>%
+  fmt_markdown(columns = c("Levels")) %>%
+  cols_label(
+    PublicNotes = "Notes"
+  ) %>%
+  # tab_header(title = "Data sharing specifications") %>%
+  tab_footnote(
+    footnote = "Level nomenclature can be cross-referenced with https://gdc.cancer.gov/resources-tcga-users/tcga-code-tables/data-levels, where 'raw' corresponses to Level 1 and 'semi-processed' most closely corresponds to Level 2.",
+    locations = cells_column_labels(columns = Levels)
+  ) %>%
+  tab_style(style = list(cell_text(color = 'gray', style = "italic")), 
+            locations = cells_body(columns = Requirement, 
+                                   rows = Requirement != "required")) %>%
+  tab_options(
+    row_group.background.color = "#FFEFDB",
+    table.font.size = "14px",
+    footnotes.marks = "letters")
+```
+
+
+## Additional considerations
+
+<!-- i.e. letter of data sharing law vs. the spirit of data sharing law -->
+The specifications should guide contributions in the "spirit of effective data sharing" and are most applicable when they lead to intended results, e.g. higher-impact shared data and reasonable balance of contributor/administrative effort given the expected value of the data shared. 
+<!-- would like to use "optimal" in place of "reasonable" someday; hard to optimize if expected values are unknown;
+are there any papers with quantitative data on what data types has actually been most reused?  -->
+But there can be debatable cases where a "required" specification is less applicable. 
+The community has anticipated issues like these and provided recommendations for some of them.
+<!-- we do want to minimize having to examine too many on a case-by-case basis, though -->
+
+### Imaging data
+
+Comments from the community noted two possible issues for imaging data. 
+
+A first question is whether imaging data should be shared when there are at most only a few images, which is of limited utility for re-analysis (e.g. with machine learning) usually requiring a large set of images.
+That is, imaging data may require a "critical mass".
```

Comment:
Just a thought exercise:
1. Do we have a sense of what that critical mass is? May be a great place to mention data sharing plan where the funding reviewers review if sufficient data is being generated for a valid experiment.
2. Lets say the community mandates 50 images as the critical mass, what is the guiding principle to mandate that anyone who is doing imaging experiments upload that many images? 
3. This may be great prompt to talk about the possibilities with transfer learning or combining datasets where even small image datasets can be of great value in the discussion section :)